### ISSUE TITLE
Login does not work with graphql@14.0.0

### ISSUE THREAD

**Issue by theodorDiaconu** [cult-of-coders/apollo#52]
Something is strange and it may be related to either GraphQl or this package or meteor-apollo-accounts the error is confusing and it refers to `tokenExpires`:

Float cannot represent non numeric value: {}

**Comment by ujwal-setlur** [comment]
Yes, I get this too. I am using phone login using ujwal:apollo-accounts and ujwal:accounts-phone. Also, I see an error message on the console when meteor starts up:

```
=> App running at: http://192.168.1.103:3000
W20180929-08:38:31.193(-7)? (STDERR) WARNING: npm peer requirements not installed:
W20180929-08:38:31.239(-7)? (STDERR)  - graphql@14.0.2 installed, graphql@0.13.x needed
W20180929-08:38:31.239(-7)? (STDERR)  - graphql-tools@4.0.0 installed, graphql-tools@3.x.x needed
W20180929-08:38:31.239(-7)? (STDERR)
W20180929-08:38:31.239(-7)? (STDERR) Read more about installing npm peer dependencies:
W20180929-08:38:31.240(-7)? (STDERR)   http://guide.meteor.com/using-packages.html#peer-npm-dependencies
W20180929-08:38:31.240(-7)? (STDERR)
```

**Comment by ujwal-setlur** [comment]
Some issues for reference:

https://github.com/orionsoft/meteor-apollo-accounts/issues/90
https://github.com/orionsoft/meteor-apollo-accounts/issues/70

**Comment by ujwal-setlur** [comment]
The following patch fixes it:

```
diff --git a/src/Auth.js b/src/Auth.js
index ff3825e..0ee8c54 100644
--- a/src/Auth.js
+++ b/src/Auth.js
@@ -7,7 +7,7 @@ type LoginMethodResponse {
   # Token of the connection
   token: String!
   # Expiration date for the token
-  tokenExpires: Float!
+  tokenExpires: String!
   # The logged in user
   user: User
 }
```

this patch also works:

```
diff --git a/src/Auth.js b/src/Auth.js
index ff3825e..d892dd0 100644
--- a/src/Auth.js
+++ b/src/Auth.js
@@ -7,7 +7,7 @@ type LoginMethodResponse {
   # Token of the connection
   token: String!
   # Expiration date for the token
-  tokenExpires: Float!
+  tokenExpires: Date!
   # The logged in user
   user: User
 }
```

Seems to indicate that whatever Meteor is sending tokenExpires as a Date to graphQL, and earlier it could parse it into a float, but anymore. Guess graphql tightened it's parser.

**Comment by smeijer** [comment]
I guess the `Date!` Only works when you've added a `Date!` scalar. I do think most people have, but still, using one of the native types would be best.

**Comment by ujwal-setlur** [comment]
That is correct. String also works.

**Comment by theodorDiaconu** [comment]
Good find @ujwal-setlur you saved me the digging time!

**Comment by theodorDiaconu** [comment]
This solved in cultofcoders:apollo-accounts@3.4.0
